src/nvim/memline.c

```diff
@@ -1032,9 +1032,7 @@ void ml_recover(bool checkext)
   line_count = 0;
   idx = 0;              // start with first index in block 1
   error = 0;
-  buf->b_ml.ml_stack_top = 0;
```

Comment:
Hmm I think resetting `size` (and this index) is a good habit when clearing the corresponding pointer. what is the _other_ assignment this is duplicate to?

Reply:
```
buf->b_ml.ml_stack_top = 0;           // nothing in the stack
```

in line 854.

Reply:
That's quite non-local. I would keep this for robustness (consider `ml_stack` an embedded struct with a coherent state).

Reply:
Shall I add an ignore directive then?